[BUG] SEO component test FAIL: not rendering correctly

### **Describe the bug**
This project contains a [SEO component](https://www.gatsbyjs.org/docs/add-seo-component/) that is not rendering correctly.

### **To Reproduce**
Steps to reproduce the behavior:
1. Switch to `dev` repository;
2. Run `npm test` in your Terminal;
3. See error messages.

### **Expected behavior**
[SEO component](https://www.gatsbyjs.org/docs/add-seo-component/) should render nothing as HTML, because the component contains Helmet components inside. All tests should pass.

### **Additional context**
The SEO component aims to put meta-tags inside `<head>` like `<title>`, favicon, meta description and OpenGraph (for sharing on Facebook and Twitter, for example).

### **Screenshots**
![image](https://user-images.githubusercontent.com/2307245/62632608-22ba7500-b909-11e9-84ca-3073092c4f3c.png)

```shell
npm test

> educodar@1.0.0 test /home/circleci/repo
> jest

  ● Console

    console.error node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9215
      The above error occurred in the <mockConstructor> component:
          in mockConstructor (created by SEO)
          in SEO
      
      Consider adding an error boundary to your tree to customize error handling behavior.
      Visit https://fb.me/react-error-boundaries to learn more about error boundaries.

  ● SEO component › renders correctly

    Invariant Violation: mockConstructor(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.

```

```js
       5 | describe("SEO component", () => {
       6 |   it("renders correctly", () => {
    >  7 |     const tree = renderer.create(<SEO />).toJSON
         |                           ^
       8 |     expect(tree).toMatchSnapshot()
       9 |   })
      10 | })
```

<details><summary>Call stack</summary>

```shell
      at invariant (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:55:15)
      at reconcileChildFibers (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:4734:13)
      at reconcileChildren (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:6500:28)
      at mountIndeterminateComponent (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7162:5)
      at beginWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7723:16)
      at performUnitOfWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11413:12)
      at workLoop (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11445:24)
      at renderRoot (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11528:7)
      at performWorkOnRoot (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12416:7)
      at performWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12328:7)
      at performSyncWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12302:3)
      at requestWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12171:5)
      at scheduleWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11986:5)
      at scheduleRootUpdate (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12585:3)
      at updateContainerAtExpirationTime (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12613:10)
      at updateContainer (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12624:10)
      at Object.create (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:13057:5)
      at Object.create (src/components/SEO/__tests__/index.js:7:27)
```

</details>

```shell
 FAIL  src/pages/__tests__/index.js
  ● Console

    console.error node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9215
      The above error occurred in the <mockConstructor> component:
          in mockConstructor (created by SEO)
          in SEO (created by HomePage)
          in HomePage
      
      Consider adding an error boundary to your tree to customize error handling behavior.
      Visit https://fb.me/react-error-boundaries to learn more about error boundaries.

  ● Home page › renders correctly

    Invariant Violation: mockConstructor(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.
```

```js
       6 | describe("Home page", () => {
       7 |   it("renders correctly", () => {
    >  8 |     const tree = renderer.create(<HomePage />).toJSON()
         |                           ^
       9 |     expect(tree).toMatchSnapshot()
      10 |   })
      11 | })
```

<details>
<summary>Call stack</summary>

```
      at invariant (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:55:15)
      at reconcileChildFibers (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:4734:13)
      at reconcileChildren (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:6500:28)
      at mountIndeterminateComponent (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7162:5)
      at beginWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7723:16)
      at performUnitOfWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11413:12)
      at workLoop (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11445:24)
      at renderRoot (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11528:7)
      at performWorkOnRoot (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12416:7)
      at performWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12328:7)
      at performSyncWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12302:3)
      at requestWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12171:5)
      at scheduleWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11986:5)
      at scheduleRootUpdate (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12585:3)
      at updateContainerAtExpirationTime (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12613:10)
      at updateContainer (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12624:10)
      at Object.create (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:13057:5)
      at Object.create (src/pages/__tests__/index.js:8:27)
````
</details>

```shell
Test Suites: 2 failed, 2 passed, 4 total
Tests:       2 failed, 2 passed, 4 total
Snapshots:   2 passed, 2 total
Time:        1.719s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
Exited with code 1
```

--- a/src/components/SEO/__tests__/index.js
+++ b/src/components/SEO/__tests__/index.js
@@ -1,6 +1,24 @@
 import React from "react"
 import renderer from "react-test-renderer"
 import SEO from ".."
+import { StaticQuery } from "gatsby"
+
+beforeEach(() => {
+  StaticQuery.mockImplementationOnce(({ render }) =>
+    render({
+      site: {
+        siteMetadata: {
+          title: "Educodar - Aulas de programação grátis para jovens",
+          titleTemplate: "%s · Educodar",
+          siteUrl: "https://educodar.com.br",
+          description:
+            "Projeto de ensino de programação básica e exposição a T.I. para estudantes de escolas de comunidades. Iniciativa sem fins lucrativos. Cursos e alimentação gratuitos.",
+          image: "/images/logo.png",
+        },
+      },
+    })
+  )
+})
 
 describe("SEO component", () => {
   it("renders correctly", () => {

--- a/src/pages/__tests__/index.js
+++ b/src/pages/__tests__/index.js
@@ -1,7 +1,13 @@
 import React from "react"
 import renderer from "react-test-renderer"
-
 import HomePage from ".."
+import { StaticQuery } from "gatsby"
+
+beforeEach(() => {
+  StaticQuery.mockImplementationOnce(({ render }) =>
+    render({ site: { siteMetadata: {} } })
+  )
+})
 
 describe("Home page", () => {
   it("renders correctly", () => {

--- a/src/pages/index.js
+++ b/src/pages/index.js
@@ -4,6 +4,7 @@ import SEO from "../components/SEO"
 const HomePage = () => (
   <>
     <SEO />
+    <h1>Página inicial</h1>
   </>
 )
 

